wit/bindgen/generator.go

```diff
@@ -817,16 +824,29 @@ func (g *generator) enumRep(file *gen.File, dir wit.Direction, e *wit.Enum, goNa
 			b.WriteRune('\n')
 		}
 		b.WriteString(formatDocComments(c.Docs.Contents, false))
-		b.WriteString(file.DeclareName(goName + GoName(c.Name, true)))
+		caseName := file.DeclareName(goName + GoName(c.Name, true))
+		b.WriteString(caseName)
 		if i == 0 {
 			b.WriteRune(' ')
 			b.WriteString(goName)
 			b.WriteString(" = iota")
 		}
 		b.WriteRune('\n')
+
+		// calling DeclareName again returns a unique name, so we need to
```

Comment:
Enums have deterministic values, so you don't need to do key: value here. Just represent it as an array:

```go
var stringsMyEnum = [5]string{
    "one",
    "two",
    "three",
    "four",
    "five",
}
```

Reply:
done